src/core/stdc/stdarg.d

```diff
@@ -459,10 +459,19 @@ else version (X86_64)
     {
     }
 
+    import core.stdc.stdlib : alloca;
+
     ///
-    void va_copy(out va_list dest, va_list src)
+    void va_copy(out va_list dest, va_list src, void* storage = alloca(__va_list_tag.sizeof))
```

Comment:
What's worrisome here is - this is a neat trick but could break easily. `alloca` is finicky and I'm note sure we have enough guarantees at the language definition level to ensure the call goes onto the callee stack.